content/recipes/coca-cola-ham.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Coca-Cola Ham
+date: 2022-11-20
+draft: false
+tags: [dinner, holiday]
+author: ["Linda Kelly"]
```

Comment:
Apparently, single quotes in double quotes still terminate the string early. Had to move the full attribution down into the page to get around this exciting limitation.